packages/create-next-app/templates/typescript/next.config.js

```diff
@@ -1,4 +1,7 @@
+// @ts-check
```

Comment:
```suggestion
```
I think we can leave the `@ts-check` off by default as we don't recommend it in our docs here https://nextjs.org/docs/api-reference/next.config.js/introduction